### ISSUE TITLE
cli: `mud test` should use top level `test` dir (rather than `src/test`)

### ISSUE THREAD

**Issue by holic** [latticexyz/mud#389]
Or really, it probably ought to figure this out via foundry config so we can consolidate on a single source of truth for configuring this?

**Comment by dk1a** [comment]
yep, this is actually what I do in #309. The config part gets a bit lost in the other changes tho

**Comment by dk1a** [comment]
I could extract the config/test changes as a separate PR if you want?
but it would still be a breaking change (everyone has `test` by default in their foundry.toml, but actual tests are in src/test according to templates etc)

**Comment by holic** [comment]
> I could extract the config/test changes as a separate PR if you want? but it would still be a breaking change (everyone has `test` by default in their foundry.toml, but actual tests are in src/test according to templates etc)

I think separating it out makes sense (I like well contained PRs) and I personally don't mind a breaking change like this (it's an easy fix for users, we should just document it).

**Comment by holic** [comment]
https://github.com/latticexyz/mud/pull/392
